src/main/java/org/tron/common/runtime/Runtime.java

```diff
@@ -176,13 +178,13 @@ public BigInteger getBlockCPULeftInUs() {
     // insure blockCap is not null
     BigInteger curBlockHaveElapsedCPUInUs =
         BigInteger.valueOf(
-            1000 * (DateTime.now().getMillis() - blockCap.getInstance().getBlockHeader()
+            Constant.ONE_THOUSAND * (DateTime.now().getMillis() - blockCap.getInstance().getBlockHeader()
                 .getRawData()
                 .getTimestamp())); // us
     BigInteger curBlockCPULimitInUs = BigInteger.valueOf((long)
-        (1000 * ChainConstant.BLOCK_PRODUCED_INTERVAL * 0.5
+        (Constant.ONE_THOUSAND * ChainConstant.BLOCK_PRODUCED_INTERVAL * 0.5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Assign this magic number 0.5 to a well-named constant, and use the constant instead. ([squid:S109](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS109))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWalyDZodPPRVdwylpM7&open=AWalyDZodPPRVdwylpM7)

[ISSUE_KEY]: AWalyDZodPPRVdwylpM7